lib/api/errors/outbound_request_forbidden.rb

```diff
@@ -35,7 +35,7 @@ class OutboundRequestForbidden < ErrorBase
       code 500
 
       def initialize(message = I18n.t("api_v3.errors.code_500_outbound_request_failure", status_code: 403))
-        super
+        super(message || I18n.t("api_v3.errors.code_500_outbound_request_failure", status_code: 403))
```

Comment:
This change looks incomplete. The default value in the method signature is the same as the fallback here, so should the default value be changed to `message = nil` for conciseness?

It's also surprising that the initializer allows overwriting the `status_code`, but we 100% ignore this.